experimental/sdk/webpubsub-socketio-extension/src/common/rest-service-client.ts

```diff
@@ -54,4 +54,12 @@ rawResponse = ${JSON.stringify(rawResponse)}`);
       getInvokeOperationSpec(this.endpoint)
     );
   }
+
+  async getClientAccessTokenUrl(options?: GenerateClientTokenOptions): Promise<string> {
+    const response = await this.getClientAccessToken(options);
+    return response.url
```

Comment:
what is this logic for?

Reply:
To get a endpoint with access_token like `https://<web-pubsub-endpoint>?access_token=XXX`
In client side, 
```
var socket = io(endpoint, { path: "/clients/socketio/hubs/eio_hub" });
```
Then the client will build connection with the correct URL `https://<web-pubsub-endpoint>/clients/socketio/hubs/eio_hub/?access_token=XXX`
Note: path string cannot be put inside endpoint, otherwise it will be recognized as namespace)